helx/charts/roger/templates/airflow-service-temp.yaml

```diff
@@ -0,0 +1,44 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ include "roger.fullname" . }}-airflow-ui-svc
+  labels:
+    app: airflow
+    component: web
+    release: {{ .Release.Name }}
+  annotations:
+    getambassador.io/config: |
+      ---
+      apiVersion: ambassador/v1
+      kind: Mapping
+      name: {{ .Values.web.ambassador.airflowUI.map_name }}
+      prefix: {{ .Values.web.ambassador.airflowUI.prefix }}
+      service: {{ include "roger.fullname" . }}-airflow-ui-svc:8080
+      ---
+      apiVersion: ambassador/v1
+      kind: Mapping
+      name: {{ .Values.web.ambassador.airflowUI.map_name }}-2
+      prefix: /static
+      service: {{ include "roger.fullname" . }}-airflow-ui-svc:8080
+      rewrite: /static
+      timeout_ms: 6000
+      ---
+      apiVersion: ambassador/v1
+      kind: Mapping
+      name: {{ .Values.web.ambassador.airflowUI.map_name }}-3
+      prefix: /admin
+      service: {{ include "roger.fullname" . }}-airflow-ui-svc:8080
+      rewrite: /admin
```

Comment:
/admin will conflict with Django's /admin path.

Reply:
This takes care of putting every thing under `/airflow` prefix ,(https://github.com/helxplatform/devops/blob/bfe61fb840ecb10d27422ca0639a4596f195a20a/helx/charts/roger/values.yaml(#L74 )

Reply:
now path is under `/airflow/admin/....`

Reply:
This is also accessible iff user is  authorized through appstore